test/test_tpm.py

```diff
@@ -91,6 +93,22 @@ def test_get_tpm_manufacturer_challenges(self):
             response = self.tpm.get_tpm_manufacturer(output=test["challenge"])
             self.assertEqual(test["response"], response)
 
+    # test parsing binary measured boot event log
```

Comment:
If you put that as a doc string, unittest will reference it if the test fails

Reply:
Right, will use doc strings for the descriptions